tests/test-ui/src/style.css

```
@@ -1,218 +1,247 @@
 
 /* Progress bar glow on the filled portion when running.
  * Stripes live on a ::after pseudo that animates transform on the compositor.
  * The pseudo is wider than the parent and translates back by one tile, so the
  * stripe pattern appears to flow without ever exposing an empty edge. */
 .progress-glow {
-  position: relative;
-  overflow: hidden;
-  box-shadow: 0 0 8px color-mix(in oklch, currentColor 40%, transparent);
+    position: relative;
+    overflow: hidden;
+    box-shadow: 0 0 8px color-mix(in oklch, currentColor 40%, transparent);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)